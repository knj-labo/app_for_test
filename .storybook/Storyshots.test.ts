import initStoryshots from "@storybook/addon-storyshots";
import { createSerializer } from '@emotion/jest';
import { puppeteerTest } from '@storybook/addon-storyshots-puppeteer';

initStoryshots();