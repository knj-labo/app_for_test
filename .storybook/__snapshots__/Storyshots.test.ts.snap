// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Components/Board Primary 1`] = `
.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-flex-wrap: wrap;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  width: 315px;
  height: 300px;
  margin: 0 auto;
}

.emotion-1 {
  width: 100px;
  height: 100px;
  border: 3px solid #555;
  font-size: 32px;
  text-align: center;
  line-height: 100px;
  cursor: pointer;
  -webkit-transition: background-color 0.3s;
  transition: background-color 0.3s;
}

.emotion-1:hover {
  background-color: #d3cce3;
}

.emotion-1:nth-of-type(-n + 3) {
  border-top: unset;
}

.emotion-1:nth-last-of-type(-n + 3) {
  border-bottom: unset;
}

.emotion-1:nth-of-type(3n) {
  border-right: unset;
}

.emotion-1:nth-of-type(3n + 1) {
  border-left: unset;
}

.emotion-10 {
  padding: 24px;
  text-align: center;
}

.emotion-11 {
  font-size: 16px;
  padding: 8px 16px;
  border: 1.5px solid #555;
  border-radius: 100px;
  background-color: transparent;
  -webkit-transition: 0.3s;
  transition: 0.3s;
  cursor: pointer;
}

.emotion-11:hover {
  background-color: #555;
  color: white;
}

<div>
  <div
    className="emotion-0"
  >
    <div
      className="emotion-1"
      onClick={[Function]}
      onKeyPress={[Function]}
      role="button"
      tabIndex={0}
    />
    <div
      className="emotion-1"
      onClick={[Function]}
      onKeyPress={[Function]}
      role="button"
      tabIndex={0}
    />
    <div
      className="emotion-1"
      onClick={[Function]}
      onKeyPress={[Function]}
      role="button"
      tabIndex={0}
    />
    <div
      className="emotion-1"
      onClick={[Function]}
      onKeyPress={[Function]}
      role="button"
      tabIndex={0}
    />
    <div
      className="emotion-1"
      onClick={[Function]}
      onKeyPress={[Function]}
      role="button"
      tabIndex={0}
    />
    <div
      className="emotion-1"
      onClick={[Function]}
      onKeyPress={[Function]}
      role="button"
      tabIndex={0}
    />
    <div
      className="emotion-1"
      onClick={[Function]}
      onKeyPress={[Function]}
      role="button"
      tabIndex={0}
    />
    <div
      className="emotion-1"
      onClick={[Function]}
      onKeyPress={[Function]}
      role="button"
      tabIndex={0}
    />
    <div
      className="emotion-1"
      onClick={[Function]}
      onKeyPress={[Function]}
      role="button"
      tabIndex={0}
    />
  </div>
  <div
    className="emotion-10"
  >
    <h3>
      次のプレイヤー
    </h3>
    <button
      className="emotion-11"
      onClick={[Function]}
      type="button"
    >
      Reset
    </button>
  </div>
</div>
`;

exports[`Storyshots Components/Board Set All Circles 1`] = `
.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-flex-wrap: wrap;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  width: 315px;
  height: 300px;
  margin: 0 auto;
}

.emotion-1 {
  width: 100px;
  height: 100px;
  border: 3px solid #555;
  font-size: 32px;
  text-align: center;
  line-height: 100px;
  cursor: pointer;
  -webkit-transition: background-color 0.3s;
  transition: background-color 0.3s;
}

.emotion-1:hover {
  background-color: #d3cce3;
}

.emotion-1:nth-of-type(-n + 3) {
  border-top: unset;
}

.emotion-1:nth-last-of-type(-n + 3) {
  border-bottom: unset;
}

.emotion-1:nth-of-type(3n) {
  border-right: unset;
}

.emotion-1:nth-of-type(3n + 1) {
  border-left: unset;
}

.emotion-10 {
  padding: 24px;
  text-align: center;
}

.emotion-11 {
  font-size: 16px;
  padding: 8px 16px;
  border: 1.5px solid #555;
  border-radius: 100px;
  background-color: transparent;
  -webkit-transition: 0.3s;
  transition: 0.3s;
  cursor: pointer;
}

.emotion-11:hover {
  background-color: #555;
  color: white;
}

<div>
  <div
    className="emotion-0"
  >
    <div
      className="emotion-1"
      onClick={[Function]}
      onKeyPress={[Function]}
      role="button"
      tabIndex={0}
    >
      O
    </div>
    <div
      className="emotion-1"
      onClick={[Function]}
      onKeyPress={[Function]}
      role="button"
      tabIndex={0}
    >
      O
    </div>
    <div
      className="emotion-1"
      onClick={[Function]}
      onKeyPress={[Function]}
      role="button"
      tabIndex={0}
    >
      O
    </div>
    <div
      className="emotion-1"
      onClick={[Function]}
      onKeyPress={[Function]}
      role="button"
      tabIndex={0}
    >
      O
    </div>
    <div
      className="emotion-1"
      onClick={[Function]}
      onKeyPress={[Function]}
      role="button"
      tabIndex={0}
    >
      O
    </div>
    <div
      className="emotion-1"
      onClick={[Function]}
      onKeyPress={[Function]}
      role="button"
      tabIndex={0}
    >
      O
    </div>
    <div
      className="emotion-1"
      onClick={[Function]}
      onKeyPress={[Function]}
      role="button"
      tabIndex={0}
    >
      O
    </div>
    <div
      className="emotion-1"
      onClick={[Function]}
      onKeyPress={[Function]}
      role="button"
      tabIndex={0}
    >
      O
    </div>
    <div
      className="emotion-1"
      onClick={[Function]}
      onKeyPress={[Function]}
      role="button"
      tabIndex={0}
    >
      O
    </div>
  </div>
  <div
    className="emotion-10"
  >
    <h3>
      次のプレイヤー
    </h3>
    <button
      className="emotion-11"
      onClick={[Function]}
      type="button"
    >
      Reset
    </button>
  </div>
</div>
`;

exports[`Storyshots Components/Board Set All Crosses 1`] = `
.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-flex-wrap: wrap;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  width: 315px;
  height: 300px;
  margin: 0 auto;
}

.emotion-1 {
  width: 100px;
  height: 100px;
  border: 3px solid #555;
  font-size: 32px;
  text-align: center;
  line-height: 100px;
  cursor: pointer;
  -webkit-transition: background-color 0.3s;
  transition: background-color 0.3s;
}

.emotion-1:hover {
  background-color: #d3cce3;
}

.emotion-1:nth-of-type(-n + 3) {
  border-top: unset;
}

.emotion-1:nth-last-of-type(-n + 3) {
  border-bottom: unset;
}

.emotion-1:nth-of-type(3n) {
  border-right: unset;
}

.emotion-1:nth-of-type(3n + 1) {
  border-left: unset;
}

.emotion-10 {
  padding: 24px;
  text-align: center;
}

.emotion-11 {
  font-size: 16px;
  padding: 8px 16px;
  border: 1.5px solid #555;
  border-radius: 100px;
  background-color: transparent;
  -webkit-transition: 0.3s;
  transition: 0.3s;
  cursor: pointer;
}

.emotion-11:hover {
  background-color: #555;
  color: white;
}

<div>
  <div
    className="emotion-0"
  >
    <div
      className="emotion-1"
      onClick={[Function]}
      onKeyPress={[Function]}
      role="button"
      tabIndex={0}
    >
      ×
    </div>
    <div
      className="emotion-1"
      onClick={[Function]}
      onKeyPress={[Function]}
      role="button"
      tabIndex={0}
    >
      ×
    </div>
    <div
      className="emotion-1"
      onClick={[Function]}
      onKeyPress={[Function]}
      role="button"
      tabIndex={0}
    >
      ×
    </div>
    <div
      className="emotion-1"
      onClick={[Function]}
      onKeyPress={[Function]}
      role="button"
      tabIndex={0}
    >
      ×
    </div>
    <div
      className="emotion-1"
      onClick={[Function]}
      onKeyPress={[Function]}
      role="button"
      tabIndex={0}
    >
      ×
    </div>
    <div
      className="emotion-1"
      onClick={[Function]}
      onKeyPress={[Function]}
      role="button"
      tabIndex={0}
    >
      ×
    </div>
    <div
      className="emotion-1"
      onClick={[Function]}
      onKeyPress={[Function]}
      role="button"
      tabIndex={0}
    >
      ×
    </div>
    <div
      className="emotion-1"
      onClick={[Function]}
      onKeyPress={[Function]}
      role="button"
      tabIndex={0}
    >
      ×
    </div>
    <div
      className="emotion-1"
      onClick={[Function]}
      onKeyPress={[Function]}
      role="button"
      tabIndex={0}
    >
      ×
    </div>
  </div>
  <div
    className="emotion-10"
  >
    <h3>
      次のプレイヤー
    </h3>
    <button
      className="emotion-11"
      onClick={[Function]}
      type="button"
    >
      Reset
    </button>
  </div>
</div>
`;
